stripe-payment-link-sms/.env

```diff
@@ -1,18 +1,17 @@
-# Configure these variables in your function configuration:
-# https://www.twilio.com/console/functions/configure
-
-
-# Find these credentials in the Twilio console
-# https://www.twilio.com/console
-ACCOUNT_SID=
-AUTH_TOKEN=
+# description: A Stripe secret key
+# format: secret
+# link: https://dashboard.stripe.com/test/apikeys
+# required: true
+STRIPE_SECRET_KEY=
 
 
-# Stripe secret key
-# https://dashboard.stripe.com/test/apikeys
-STRIPE_SECRET_KEY=
+# description: Sender phone number. Default is "STRIPEDEMO"
```

Comment:
maybe strip out the "default is STRIPEDEMO" if we aren't setting it?

Reply:
It's set in the code as fallback.